docs/CMIP7/guidance_for_users.md

```diff
@@ -5,11 +5,233 @@ title: CMIP7 Guidance for Data Users
+
+
+
+
+## 3. CMIP7 facets and their documentation
+
+CMIP7 datasets can be identified through a series of facets that represents key attributes of the data. The facets are:
+
+* activity
+* institution
+* source
+* experiment
+* variant
+* realm
+* frequency
+* variable
+* grid
+* version
+
+
+The values associated with each facets are standardized through the [CVs][cmipCvs]. They are used to search the ESGF database and can be found in the global attributes of the data. This section provides helpful links and gives a bit more information on a few key facets. 
+
+
+### 3.1.  Source and Variant
+* [List of models][sourceIdhtml]
+* [Essential Model Documentation (EMD)][emd]
+
+The Essential Model Documentation (EMD) contains high-level description intended to contain information on model formulation that can be easily compared between different models. EMD pages contain links to more in-depth model documentation for each source.
+
+The source facet gives the name of the model and the variant facet represents each member of an ensemble for a given source. It can also be called the “ripf” identifier (“r” for realization, “i” for initialization, “p” for physics, and “f” for forcing).
+
+A useful tool to assess the models is the [Rapid Evaluation Framework (REF)][ref]. It is an assessment of the models participating in the CMIP7 Assessment Fast Track (AFT). 
+
+
+### 3.2.  Experiment and Activity
+* [List of experiments][experimentIdhtml]
+* [List of activities][activityIdJson]
+* TODO: Does experiment doc exist for CMIP7 ?
+
+ 
+The CMIP7 protocol and experiments are described in a [special issue][GMDSpecialIssue] of Geoscientific Model Development with an overview of the design and scientific strategy provided in the lead article of that issue by [Dunne et al. (2025)][dunne2025].
+
+Each model participating in CMIP7 will contribute results from the eight DECK experiments. These experiments are the only ones directly overseen by the [CMIP Panel][CMIPPanel], and together these constitute the ongoing (slowly evolving) “CMIP” activity. In addition to the DECK, each modeling group may choose to contribute to any of the [CMIP7 endorsed MIPs][CMIPEndorsedMips]. The CMIP panel identifies key experiments to be prioritized on different timelines through fast tracks. The first one is the AFT.
+
+
+### 3.3. Variable
+* [List of variables][variableid]
+* [Branded variable documentation](branded_variable.md)
+
+The variables produced in CMIP7 were recommended by the [CMIP7 Data Request task team][cmipDataRequest]. In CMIP7, the concept of branded variable uniquely identifies each variable. It follows the  template: 
```

Comment:
This has ended up being a bit messier. My two cents: just point at the DR and focus on that. I don't think we need to figure out whether branded variable actually provides a unique ID or not to still provide good advice.